From 721c51a780a74f914e5a846ad1e503a483347c89 Mon Sep 17 00:00:00 2001
From: Othmane AHL ZOUAOUI <othmane.ahlzouaoui@st.com>
Date: Wed, 31 Jul 2024 10:49:35 +0200
Subject: [PATCH 2/8] onnxruntime: add SONAME with MAJOR version

Signed-off-by: Othmane AHL ZOUAOUI <othmane.ahlzouaoui@st.com>
---
 cmake/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/cmake/CMakeLists.txt b/cmake/CMakeLists.txt
index 8edbb6ad6f..2a18095d00 100644
--- a/cmake/CMakeLists.txt
+++ b/cmake/CMakeLists.txt
@@ -1801,6 +1801,9 @@ if (onnxruntime_ENABLE_EXTERNAL_CUSTOM_OP_SCHEMAS)
   )
 endif()

+message("ONNXRUNTIME_VERSION_MAJOR=${ONNXRUNTIME_VERSION_MAJOR}")
+set_target_properties(onnxruntime PROPERTIES SOVERSION "${ONNXRUNTIME_VERSION_MAJOR}")
+
 if(TARGET onnxruntime)
 # Install
   include(GNUInstallDirs)
--
2.34.1
