From e012f7997b4cf389593c04527785c8201f7d6834 Mon Sep 17 00:00:00 2001
From: Vincent ABRIOU <vincent.abriou@st.com>
Date: Wed, 1 Jun 2022 09:56:28 +0200
Subject: [PATCH 4/4] add glog library dependency

Signed-off-by: Vincent ABRIOU <vincent.abriou@st.com>
---
 makefile_build/Makefile | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/makefile_build/Makefile b/makefile_build/Makefile
index 9dce39e..9193ae0 100644
--- a/makefile_build/Makefile
+++ b/makefile_build/Makefile
@@ -52,6 +52,9 @@ LIBCORAL_CXXFLAGS := \
 # Make sure only MPL2.0 or more permissively licensed code is included.
 LIBCORAL_CXXFLAGS += -DEIGEN_MPL2_ONLY
 
+LIBCORAL_LDFLAGS := \
+	-lglog
+
 LIBDIR := $(BUILDDIR)/lib
 LIBCORAL_NAME := libcoral.so.$(LIBCORAL_VERSION)
 LIBCORAL_PATH := $(LIBDIR)/$(LIBCORAL_NAME)
@@ -65,7 +68,7 @@ $(LIBCORAL_CCOBJS): $(BUILDDIR)/%.o: %.cc
 
 $(LIBCORAL_PATH): $(LIBCORAL_CCOBJS)
 	@$(MKDIR)
-	$(CXX) -shared $^ -o $@ -Wl,-soname,$(LIBCORAL_NAME_MAJOR)
+	$(CXX) -shared $^ -o $@ $(LIBCORAL_LDFLAGS) -Wl,-soname,$(LIBCORAL_NAME_MAJOR)
 
 clean:
 	rm -rf $(BUILDDIR)
-- 
2.25.1

