From 2207968a073b0633f99d8e4aaa57ffb78cce7e64 Mon Sep 17 00:00:00 2001
From: Othmane <othmane.ahlzouaoui@st.com>
Date: Tue, 2 Jun 2020 09:58:41 +0200
Subject: [PATCH 1/1] Add sparsity to Makefile sources

---
 tensorflow/lite/tools/make/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tensorflow/lite/tools/make/Makefile b/tensorflow/lite/tools/make/Makefile
index 4219aa5c31..43ded75fd6 100644
--- a/tensorflow/lite/tools/make/Makefile
+++ b/tensorflow/lite/tools/make/Makefile
@@ -110,6 +110,7 @@ $(wildcard tensorflow/lite/kernels/*.cc) \
 $(wildcard tensorflow/lite/kernels/internal/*.cc) \
 $(wildcard tensorflow/lite/kernels/internal/optimized/*.cc) \
 $(wildcard tensorflow/lite/kernels/internal/reference/*.cc) \
+$(wildcard tensorflow/lite/tools/optimize/sparsity/*.cc) \
 $(PROFILER_SRCS) \
 tensorflow/lite/tools/make/downloads/farmhash/src/farmhash.cc \
 tensorflow/lite/tools/make/downloads/fft2d/fftsg.c \
-- 
2.17.1

