From 638de2e65936dd8d012ce19a090186542f15637a Mon Sep 17 00:00:00 2001
From: Maxence GUILHIN <maxence.guilhin@st.com>
Date: Fri, 27 Sep 2024 16:23:24 +0200
Subject: [PATCH 2/2] fix uncompatible cmake flag issue

Signed-off-by: Maxence GUILHIN <maxence.guilhin@st.com>
---
 cmake/onnxruntime_mlas.cmake | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/cmake/onnxruntime_mlas.cmake b/cmake/onnxruntime_mlas.cmake
index c02ac2096d..1bbe1120b6 100644
--- a/cmake/onnxruntime_mlas.cmake
+++ b/cmake/onnxruntime_mlas.cmake
@@ -362,6 +362,17 @@ else()
         set_source_files_properties(${MLAS_SRC_DIR}/sqnbitgemm_kernel_neon_int8.cpp
                                     PROPERTIES COMPILE_FLAGS " -march=armv8.2-a+dotprod")
         if (NOT APPLE)
+          string(REGEX REPLACE "-mcpu=[-a-zA-Z0-9_.^$*+?]*" "" _tmp ${CMAKE_C_FLAGS})
+          set(CMAKE_C_FLAGS ${_tmp})
+          message("CMAKE_C_FLAGS: ${CMAKE_C_FLAGS}")
+
+          string(REGEX REPLACE "-mcpu=[-a-zA-Z0-9_.^$*+?]*" "" _tmp ${CMAKE_ASM_FLAGS})
+          set(CMAKE_ASM_FLAGS ${_tmp})
+          message("CMAKE_ASM_FLAGS: ${CMAKE_ASM_FLAGS}")
+
+          string(REGEX REPLACE "-mcpu=[-a-zA-Z0-9_.^$*+?]*" "" _tmp ${CMAKE_CXX_FLAGS})
+          set(CMAKE_CXX_FLAGS ${_tmp})
+          message("CMAKE_CXX_FLAGS: ${CMAKE_CXX_FLAGS}")
           set(mlas_platform_srcs
             ${mlas_platform_srcs}
             ${MLAS_SRC_DIR}/aarch64/HalfGemmKernelNeon.S
-- 
2.25.1

